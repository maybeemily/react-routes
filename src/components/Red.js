import React from 'react';

export default function Red() {
  const style = {
    backgroundColor: 'red',
    width: '50vw',
    height: '50vw',
    fontSize: '4em',
    letterSpacing: '1em',
    color: 'black'
  };

  return (
    <div style={style}>RED</div>
  );
}
