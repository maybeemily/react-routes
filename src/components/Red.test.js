import React from 'react';
import { shallow } from 'enzyme';
import Red from './Red';

describe('Red component', () => {
  it('renders red div', () => {
    const wrapper = shallow(<Red/>);
    expect(wrapper).toMatchSnapshot();
  });
});
