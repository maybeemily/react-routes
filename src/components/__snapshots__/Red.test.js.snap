// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Red component renders red div 1`] = `
<div
  style={
    Object {
      "backgroundColor": "red",
      "color": "black",
      "fontSize": "4em",
      "height": "50vw",
      "width": "50vw",
    }
  }
>
  R E D
</div>
`;
